### DISCUSSION TITLE
Valid gltf models not loading in Scene Viewer

### DISCUSSION THREAD

**Discussion by ukito-pl** [google/model-viewer#3644] [Q&A]
### Description

Sometimes model that is valid according to offical Khronos gltf validator doesn't load into SceneViewer or in online preview tool https://arvr.google.com/scene-viewer-preview
Here is example model that fails to load in scene viewer: 
[test.zip](https://github.com/google/model-viewer/files/5415013/test.zip)
In console I can see following error:
![image](https://user-images.githubusercontent.com/19152578/96706915-7b9fed00-1397-11eb-9156-906f7f9793e1.png)

Unfortunately I'm not sure how to interpret this error, I am not familiar with term 'geometry merging' in gltf context. Does anyone have any clue why this happens and what is wrong with attached gltf file?

**Comment by elalish** [comment]
@tpsiaki Looks like another SceneViewer glTF loading bug?
